integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/AbstractBridge.java

```diff
@@ -0,0 +1,68 @@
+package com.redhat.service.bridge.integration.tests.common;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.keycloak.representations.AccessTokenResponse;
+
+import com.redhat.service.bridge.test.resource.KeycloakResource;
+
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+public abstract class AbstractBridge {
```

Comment:
Is there a need to have this as an abstract class?
I think this can be used as an utility class, providing token.

Reply:
I added this as an abstract class because make reuse of variables declared.
Since we are declaring in maven parameters, i will move this to TokenUtil class